pages/tasks/task.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I've refactored into `pages/tasks/regions' and squashed the commits.  Hopefully, it is all good to go now! :smile: